packages/core/src/Configs/MintV3PoolConfig.class.ts

```diff
@@ -82,14 +89,31 @@ export class MintV3PoolConfig extends Config<IMintV3PoolConfigArgs> {
   validate(): void {
     super.validate();
 
-    if (!this.fee) {
-      throw new Error(`No fee was set, but is required.`);
+    if (!this.fees) {
+      throw new Error(`No fees were set, but are required.`);
     }
 
-    if (this.fee > MintV3PoolConfig.MAX_FEE) {
-      throw new Error(
-        `Fees cannot supersede the max fee of ${MintV3PoolConfig.MAX_FEE}.`
-      );
+    if (this.fees.ask === this.fees.bid) {
+      if (
+        this.fees.ask > MintV3PoolConfig.MAX_FEE ||
+        this.fees.bid > MintV3PoolConfig.MAX_FEE
+      ) {
+        throw new Error(
+          `Fees cannot supersede the max fee of ${MintV3PoolConfig.MAX_FEE}.`
+        );
+      }
+    } else {
+      if (this.fees.ask > MintV3PoolConfig.MAX_FEE) {
+        throw new Error(
+          `Ask fee cannot supersede the max fee of ${MintV3PoolConfig.MAX_FEE}.`
+        );
+      }
+
+      if (this.fees.bid > MintV3PoolConfig.MAX_FEE) {
+        throw new Error(
+          `Take fee cannot supersede the max fee of ${MintV3PoolConfig.MAX_FEE}.`
+        );
+      }
```

Comment:
Validation to ensure that the user doesn't set a fee higher than the protocol's max.

Reply:
What is that constant now? 5%?